### PR TITLE
Added option to add tags/usecase to approved list and packs metadata in same PR

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37364

## Description
Added option to add tags/usecase to approved list and packs metadata in same PR

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * Adding the option to run **validate** on a specific path while using git (*-i* & *-g*).
 * The **format** command will now change UUIDs in .yml and .json files to their respective content entity name.
 * Added a playbook validation to check if a task sub playbook exists in the id set in the **validate** command.
+* Added the option to add tags/usecases to approved list and packmetadata in same pull request.
```
Comment (dantavori) [inline CHANGELOG.md:7]:
```suggestion
* Added the option to add new tags/usecases to the approved list and to the pack metadata on the same pull request.
```

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 2]
```diff
@@ -24,7 +24,9 @@
 from demisto_sdk.commands.common.errors import Errors
 from demisto_sdk.commands.common.hook_validations.base_validator import \
     BaseValidator
-from demisto_sdk.commands.common.tools import (get_core_pack_list, get_json,
+from demisto_sdk.commands.common.tools import (get_core_pack_list,
+                                               get_current_tags,
+                                               get_current_usecases, get_json,
```
Comment (dantavori) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:29]: no need, there is already tools imported in this file

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 3]
```diff
@@ -388,7 +390,9 @@ def _is_approved_usecases(self) -> bool:
         try:
             approved_usecases = tools.get_approved_usecases()
             pack_meta_file_content = json.loads(self._read_file_content(self.pack_meta_file))
-            non_approved_usecases = set(pack_meta_file_content[PACK_METADATA_USE_CASES]) - set(approved_usecases)
+            current_usecases = get_current_usecases()
```
Comment (dantavori) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:393]:
```suggestion
            current_usecases = tools.get_current_usecases()
```

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 4]
```diff
@@ -408,7 +412,8 @@ def _is_approved_tags(self) -> bool:
         try:
             approved_tags = tools.get_approved_tags()
             pack_meta_file_content = json.loads(self._read_file_content(self.pack_meta_file))
-            non_approved_tags = set(pack_meta_file_content[PACK_METADATA_TAGS]) - set(approved_tags)
+            current_tags = get_current_tags()
```
Comment (dantavori) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:415]:
```suggestion
            current_tags = tools.get_current_tags()
```

**Review by dantavori — CHANGES_REQUESTED** [review]
awesome, fix the following minor notes and we can merge 💣

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py (hunk 2 above)
Comment (daryakoval) [reply demisto_sdk/commands/common/hook_validations/pack_unique_files.py:29]: removed functions that I added

**Review by dantavori — APPROVED** [review]

**Pull request merged**